### PR TITLE
Update DEVELOPERS.md

### PR DESCRIPTION
**Pull request by gargarchit**
I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.
Please merge this
accordproject/techdocs#226

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ machine:
 * [Git][git]: The [Github Guide to Installing Git][git-setup] is a good source of information.
 
 * [Node.js v10.16.0 'Dubnium' (LTS)][node]: We use Node to generate the documentation, run a
-  development web server, run tests, and generate distributable files. Depending on your system,
+  development of web server, run tests and generate distributable files. Depending on your system,
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ To contribute code to Cicero, you must have a GitHub account so you can push cod
 fork of Cicero and open Pull Requests in the [GitHub Repository][github].
 
 To create a Github account, follow the instructions [here][github-signup].
-Afterwards, go ahead and [fork][github-forking] the
+Afterward, go ahead and [fork][github-forking] the
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ We have very precise rules over how our git commit messages can be formatted.  T
 readable messages** that are easy to follow when looking through the **project history** and **git logs**.  
 But also, we use the git commit messages to **generate the Cicero change log**.
 
-The commit message formatting can be added using a version of typical git workflow.
+The commit message formatting can be added using a version of the typical git workflow.
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 4]
```diff
@@ -154,10 +154,10 @@ Formatting for the body is displayed in this example:
 
 ### Changes
 - Change one
-  - Subchange one
-  - Subchange two
+  - Sub change one
+  - Sub change two
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

**Comment by jeromesimeon** [comment]
No activity on this. I'm closing the PR.

**Pull request closed without merging**